SparkPost/2web2ui
# FE-877 - Component wrapper for matchbox Modal

FE-877 - Component wrapper for matchbox Modal

### What Changed
 - Added a component wrapper for Modal
 - replaced all imports from @sparkpost/matchbox with src/component/matcbox
 - moved the previous global Modal component in src/component/matcbox/Modal and also replaced all its previous imports.

### How To Test
 - Verify Update Payment and Billing contact modal on /account/billing.
 - Verify Support Modal.
 - Verify Feature Comparison modal on /account/billing/plan.
 - Verify all the imports have been replaced.

### To Do
